src/LendingPool.sol

```diff
@@ -1027,25 +1027,25 @@ contract LendingPool is LendingPoolGuardian, Creditor, DebtToken, InterestRateMo
         }
     }
 
-    function startLiquidation(address account)
+    function startLiquidation()
         external
+        override
         onlyAccount
         whenLiquidationNotPaused
         processInterests
         returns (uint256 startDebt)
     {
-        // TODO: delete address acoount and refactor to the msg.sender
         //Only Accounts can have debt, and debtTokens are non-transferrable.
```

Comment:
We can check if debt is non-zero instead using the "onlyAccount" modifier?
(As said in the comments)